lib/bitcoin/BitcoinBlockDataIterator.ts

```diff
@@ -1,6 +1,7 @@
 import BitcoinBlockModel from './models/BitcoinBlockModel';
 import BitcoinFileReader from './BitcoinFileReader';
 import BitcoinRawDataParser from './BitcoinRawDataParser';
+import logger from '../common/Logger';
```

Comment:
Curious about casing: any reason to use lower case `logger` instead of `Logger`?

Reply:
Trying to use the same pattern like `console.info()` (e.g. `logger.info()`). i.e. `Logger` is a class, `logger` is an instance.

Reply:
Took Ali's suggestion.